src/agents/llama_guard.py

```diff
@@ -76,13 +77,11 @@ def parse_llama_guard_output(output: str) -> LlamaGuardOutput:
 
 class LlamaGuard:
     def __init__(self) -> None:
-        if os.getenv("GROQ_API_KEY") is None:
+        if settings.GROQ_API_KEY is None:
             print("GROQ_API_KEY not set, skipping LlamaGuard")
             self.model = None
             return
-        self.model = ChatGroq(model="llama-guard-3-8b", temperature=0.0).with_config(
-            tags=["llama_guard"],
-        )
+        self.model = get_model(GroqModelName.LlAMA_GUARD_3_8B)
```

Comment:
I don't think we should hide the `with_config` inside of get_model, it's too easy to miss for this case. Can we keep it explicit in this file instead?

Reply:
fixed